src/main/java/base/navigationservice/KeyboardListener.java

```diff
@@ -26,6 +26,9 @@ public void keyPressed(KeyEvent event) {
         if (keys[KeyEvent.VK_CONTROL] && keys[KeyEvent.VK_S]) {
             game.handleCTRLandS();
         }
+        if(keys[KeyEvent.VK_M]){ //press M
+            game.defualtMap();    //replace map with default
```

Comment:
one more thing: 
comments can be removed from code as long as we have meaningful names. So instead of leaving a comment 'replace map with default' we can name a method 'replaceMapWithDefault'